### PR TITLE
Use pr-all-count

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Note: I am not sure why but it still displays the number of released PRs. Please help to finish this :smile:

### CODE REVIEW

File: src/js/components/pipeline/MainMetrics.jsx [hunk 1]
```diff
@@ -37,8 +37,8 @@ const MainMetrics = ({ data, loading, spinner }) => {
           <div className="col-md-3">
             <MainMetric
               title="Pull requests"
-              hint="Number of pull requests released, counting in the Lead Time."
-              dataGetter={() => [data.releasedPRs.avg, data.releasedPRs.variation]}
+              hint="Number of mentioned pull requests."
```
Comment (warenlg) [inline src/js/components/pipeline/MainMetrics.jsx:40]:
```suggestion
              hint="Number of pull requests engaged during the time interval."
```

**Comment by vmarkovtsev** [comment]
ok, the product is happy, then @se7entyse7en can you or @dpordomingo test this, please?

**Comment by dpordomingo** [comment]
@warenlg @vmarkovtsev I see a strange thing if you compare both screenshots; you can see that:
- (from the 1st. image) getting the whole period between `2016-12-01` and `2016-12-15`, the PR count is `54`, which is OK considering the list of PRs returned by `/filter/pull_requests` for the same period.
- (from the 2nd. image) getting the same data for the extended period (same plus the previous one: `2016-11-16` and `2016-12-15`), and aggregating by `15 days`, and considering the second group (between `2016-12-01` and `2016-12-15`, the same as above), the PR count is `36` instead of the expected `54`.

Imo both should be the same.

**Comment by vmarkovtsev** [comment]
It's cool that you noticed this David :+1:  I need a JIRA issue to dig this.

**Comment by dpordomingo** [comment]
here is the issue:
https://athenianco.atlassian.net/browse/ENG-752

**Pull request merged**